### DISCUSSION TITLE
Model fitting when both have same AIC value

### DISCUSSION THREAD

**Discussion by Zehidul-EcoMove** [bmcclintock/momentuHMM#177] [Q&A] [answered]
Hi,
I have been using the package to analyse large carnivore movement behaviour and while running sets of models, I have similar two models with the same AIC value. In this case how to deal with the situation, either we do model averaging or there are any other way??

**Comment by bmcclintock** [comment, marked as answer]
It's pretty unusual for two distinct models (similar or not) to have the exact same AIC value. I'd suggest double checking that: 1) the models are specified correctly; 2) all parameters are estimable; 3) any covariates in the models are not highly correlated; and 4) the likelihoods have converged to (what appears to be) the global maximum. It may be more informative to focus on the negative log-likelihood value (instead of AIC) when trying to assess potential issues. This can be accessed via `m$mod$minimum`, where `m` is a fitted `momentuHMM` model object. 

Assuming it's not one of these (or any other model-fitting issue), I'd suggest consulting the AIC literature on potential avenues for dealing with the situation. [Burnham & Anderson](https://link.springer.com/book/10.1007/b97636) would be a good start.
